dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportCLITool.java

```diff
@@ -48,348 +47,276 @@
  * Modified by David Little, UCSD Libraries 12/21/04 to
  * allow the registration of files (bitstreams) into DSpace.
  */
-public class ItemImportCLITool {
-
-    private static boolean template = false;
+public class ItemImportCLITool extends DSpaceRunnable<ItemImportScriptConfiguration> {
```

Comment:
Same here, you'll need to split this into two classes... likely `ItemImportCLI` and ItemImport`.  See the `org.dspace.app.bulkedit.` classes as examples (`MetadataImport` and `MetadataImportCLI`).